docs/authentication/configuration/sign-up-sign-in-options.mdx

```diff
@@ -51,17 +51,55 @@ Clerk offers two kinds of authentication strategies: **password** and **password
 
 Choosing the **password** strategy requires users to set a password during the sign up process. Clerk offers out of the box protection against weak and leaked passwords and the only requirement enforced is that the password be a miminum of 8 characters. (**Note**: passwordless authentication is still available to users even if password strategy is selected.)
 
-There are two types of **passwordless** authentication strategies:
+Choosing a **passwordless** strategy provides users with a more secure way to sign up and sign in to your application. They are also more convenient than passwords because they don't require users to remember a complex string of characters.
 
-- **One-time password (OTP)**
-- **Email link**
+**Passwordless** authentication strategies include:
+
+- **[Passkeys](#passkeys)**
+- **[One-time password (OTP)](#one-time-password-otp)**
+- **[Email link](#email-link)**
 
 To configure authentication strategies:
 
 1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) and select your application.
 2. In the navigation sidebar, select **User & Authentication > Email, Phone, Username**.
 3. In the **Authentication strategies** section, toggle on the authentication strategies you would like to enable.
 
+### Passkeys
+
+A passkey is a type of sign-in credential that requires one user action, but uses two authentication factors:
+  1. A pin number or biometric data
+  1. A physical device
+
+Because passkeys use two authentication factors, they are the most secure **passwordless** authentication strategy.
+
+Users can only create passkeys after they have signed up, which means you will need to enable another authentication strategy for the sign-up process. Once a user is signed in, they can create a passkey.
+
+<Callout type="Info">
+  Passkey authentication is a premium feature and is not available on the Free plan. [Upgrade your plan](https://clerk.com/pricing) to enable this feature.
+</Callout>
+
+#### Manage user passkeys
+
+The easiest way to allow your users to create and manage their passkeys is to use the prebuilt [`<UserProfile>`](/docs/components/user/user-profile) component, which includes passkey management in the **Security** tab.
+
+If you are building your a custom user interface, check out the [passkeys custom flow](/docs/custom-flows/passkeys) guide to learn how to build a custom passkey management flow using Clerk's API.
+
+#### Passkey limitations
+
+- Passkeys are not currently available as an [MFA](#multi-factor-authentication) option.
```

Comment:
so for this hash to work, you would have to change the heading of "Set up multi-factor authentication" to just "Multi-factor authentication", like I suggested here: https://github.com/clerk/clerk-docs/pull/811/files#r1550298833